docs/spell_bonus_data.md

```diff
@@ -91,6 +91,6 @@ If > 0
 
 Use this as new attack power coefficient.
 
-### comments
+### triggerDummy
```

Comment:
this is wrong. You are not editing any DB stuff you are just adding functions. revert this